layouts/shortcodes/diagram_variants.svg

```diff
@@ -485,7 +485,7 @@ $(document).ready(function() {
          sodipodi:role="line"
          x="29.392647"
          y="32.938339"
-         id="tspan5561">VMWare VSphere</tspan></text>
+         id="tspan5561">VMware VSphere</tspan></text>
```

Comment:
Since we're fixing upper/lower case: VMware spells it "vSphere". :grinning:

Reply:
Great catch!